Updates.md

```diff
@@ -0,0 +1,87 @@
+    4. If _selectedVersion_ is `null` or _selectedVersion_ is less than or equal to _version_, set _selectedVersion_ to _version_ and _selectedUrl_ to _url_. 
+4. If _selectedVersion_ is `null`, abort these steps.
+5. If _selectedVersion_ is less than or equal to the currently installed version, abort these steps.
+6. Fetch the resource from _selectedUrl_.
+7. If the resource is not a valid Signed Web Bundle, abort these steps.
+8. If the Web Bundle ID of the resource is not equal to that of the currently installed app, abort these steps.
+9. Load `/manifest.webmanifest` from the Web Bundle and parse it as a Web Application Manifest.
+10. If the resource is not found or fails to parse, abort these steps.
+11. If the `version` field in the manifest from the Web Bundle is not equal to _selectedVersion_, abort these steps.
+12. Replace the Web Bundle for the current installed app with the downloaded resource.
+
+## Extensions to the Web Application Manifest
+
+### `version` field
+
+The optional `version` field contains a string indicating the version of the application. For the purposes of updating version numbers only need to be ordered. For example, a plain integer versioning scheme (e.g. 1, 2, 3, etc.) would be sufficient however for ease of use and to provide for more complex comparisons to be performed on versions a structured format such as “A.B.C.D”, where A, B, C and D are positive integers (or zero) is much more useful. A format such as [Semantic Versioning](https://semver.org/) could also be adopted as it provides even more expressivity.
+
+Whatever the structure of the version string, a comparison algorithm must be defined so that sorting of and comparisons between versions can be performed in the algorithms defined in this specification. Lexical comparison between version strings is inappropriate. A manifest without the `version` field set should be treated as having the lowest-possible version.
+
+### `update_manifest_url` field
+
+The optional `update_manifest_url` field indicates the location where the Web Application Update Manifest can be found. It must be an absolute HTTPS URL, or for testing purposes a localhost HTTP URL.
+
+## Web Application Update Manifest
+
+The Web Application Update Manifest is a document listing all application versions available for download. It consists of a single JSON object with a single field called `versions` which is a list. Each entry in the list is an object containing two fields, `version` and `url`. The `version` field is a string in the same format as the proposed Web Application Manifest `version` field. The `url` field is a URL and is resolved relative to the location of the manifest. It must be an HTTPS URL or a localhost HTTP URL, for testing purposes. For example a manifest hosted at `https://developer.example.com/app/updates.json` could contain,
+
+```
+{
+  "versions": [
+    {
+      "version:" "5.2.17",
+      "src": "https://cdn.example.com/app-package-5.2.17.swbn"
```

Comment:
The text calls this the `url` field, while the example update manifest calls this the `src` field.